CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Updog Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
```

Comment:
Correct me if wrong, but I don't think we've discussed Project Maintainer as a title within the team. It could be worth clarifying this part both in documentation and with the team. Are we all "Project Maintainers"? Are "Product Managers" doubling as "Project Maintainer"s?

Reply:
I broadly interpreted us all as project maintainers, and specifying the structure too much might complicate the handover to the next team working on the project in case they do things differently. What are your thoughts?

Reply:
Good points, there'll always be bits where we aren't able to behave exactly like an open source project.